### PR TITLE
47198_master_layout

### PR DESCRIPTION
**Pull request by Omid07**
screenshot from 2015-12-01 14 53 13

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'debug' => env('APP_DEBUG', false),
+    'debug' => env('APP_DEBUG', true),
```
Comment (asifbuetcse) [inline config/app.php]: change this setting to false.

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                  @else
+                     <li class="navbar-text"><i class="fa fa-btn fa-user"></i>{{ Auth::user()->name }}</li>
+                     <li><a href="/auth/logout"><i class="fa fa-btn fa-sign-out"></i>Logout</a></li>
+                  @endif
+               </ul>
+             </div>
+         </nav>
+      </div>
+   </div>
+    @yield('content')
```
Comment (asifbuetcse) [inline resources/views/layouts/app.blade.php]: Check indentation of @yield

**Comment by asifbuetcse** [comment]
Ok.

File: config/app.php [hunk 3]
```diff
@@ -193,6 +194,8 @@
         'URL'       => Illuminate\Support\Facades\URL::class,
         'Validator' => Illuminate\Support\Facades\Validator::class,
         'View'      => Illuminate\Support\Facades\View::class,
+        'Form' => Collective\Html\FormFacade::class,
```
Comment (dinhhoanglong91) [inline config/app.php]: Can you order `=>`. It looks better.
```php
'View'      => Illuminate\Support\Facades\View::class,
'Form'     => Collective\Html\FormFacade::class,
```

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    {!! Html::style('css/stylesheet.css') !!}
+</head>
+<body>
+   <div class="row">
+      <div class="col-xs-12">
+         <nav class="navbar navbar-inverse">
+            <ul><a class="navbar-brand" href="#"><img src="logo.png"></a></ul>
```
Comment (dinhhoanglong91) [inline resources/views/layouts/app.blade.php]: Move this image to `public/img/logo.png`

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**